aiida_quantumespresso/calculations/opengrid.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+"""`CalcJob` implementation for the open_grid.x code of Quantum ESPRESSO."""
+from aiida.orm import RemoteData, FolderData, KpointsData
+from aiida_quantumespresso.calculations.namelists import NamelistsCalculation
+
+
+class OpengridCalculation(NamelistsCalculation):
+    """`CalcJob` implementation for the open_grid.x code of Quantum ESPRESSO."""
+
+    _default_namelists = ['INPUTPP']
+    _blocked_keywords = [
+        ('INPUTPP', 'outdir', NamelistsCalculation._OUTPUT_SUBFOLDER),
+        ('INPUTPP', 'prefix', NamelistsCalculation._PREFIX),
+        ('INPUTPP', 'overwrite_prefix', True),
```

Comment:
Since [that commit](https://gitlab.com/QEF/q-e/-/commit/f0e9e696f53ad36c5c1f7d70369387a5106bf9a5) has been included since QE 7.0, so I assume it is not a problem for aiida-qe now^^

Reply:
Well, actually 😅 

https://github.com/aiidateam/aiida-quantumespresso#compatibility-matrix

Reply:
That said, if you can't think of a reasonably easy workaround to also support QE>=6.6, I think it's better to merge the PR than stand on ceremony.